sparkles/core.py

```diff
@@ -237,19 +238,30 @@ def stylize(text, category):
 def get_acas_from_pickle(load_name, loud=False):
     """Get dict of proseco ACATable pickles for ``load_name``
 
-    Note that ``load_name`` can be a Table with columns ``obsid``
-    and ``aca`` (ACATable catalog object).
+    ``load_name`` can be a full file name (ending in .pkl or .pkl.gz) or any of the
+     following, which are tried in order:
 
-    :param load_name: load name or Table (see preview_load() doc for details)
+    - <load_name>_proseco.pkl.gz
+    - <load_name>.pkl.gz
+    - <load_name>_proseco.pkl
+    - <load_name>.pkl
+
+    :param load_name: load name
     :param loud: print processing information
     """
-    filenames = [load_name, f'{load_name}_proseco.pkl', f'{load_name}.pkl']
+    filenames = [load_name,
+                 f'{load_name}_proseco.pkl.gz', f'{load_name}.pkl.gz',
+                 f'{load_name}_proseco.pkl', f'{load_name}.pkl']
     for filename in filenames:
         pth = Path(filename)
-        if pth.exists() and pth.is_file() and pth.suffix == '.pkl':
+        if pth.exists() and pth.is_file() and pth.suffixes in (['.pkl'], ['.pkl', '.gz']):
```

Comment:
This looks great.  I have less experience with pth.suffixes, but this seems to hit the use cases.